server/dals/expression-dal.js

```diff
@@ -144,19 +141,21 @@ module.exports = {
         return sequelize.query(buildExpressionQuery(req.query),
                 { type: sequelize.QueryTypes.SELECT })
                     .then(function (stdname) {
+                        const convertToJSON = {
+                            "DegradationRates" : convertProductionDegradationRateToJSON(stdname, "degradation_rate"),
+                            "ProductionRates" : convertProductionDegradationRateToJSON(stdname, "production_rate"),
+                            "ExpressionData" : convertExpressionToJSON(
+                                stdname,
+                                req.query.dataset,
+                                expressionTimepointsByDataset[req.query.dataset],
+                                req.query.genes.split(",")
+                            )
+                        };
                         const type = req.query.type;
-                        if (type === "DegradationRates") {
-                            return res.send(convertProductionDegradationRateToJSON(stdname, "degradation_rate"));
-
-                        } else if (type === "ProductionRates") {
-                            return res.send(convertProductionDegradationRateToJSON(stdname, "production_rate"));
-
-                        } else if (type === undefined) {
-                            let dataset = req.query.dataset;
-                            let geneList = req.query.genes.split(",");
-                            let response = convertExpressionToJSON(
-                                stdname, dataset, expressionTimepointsByDataset[dataset], geneList);
-                            return res.send(response);
+                        if (Object.keys(convertToJSON).includes(type)) {
+                            return res.send(convertToJSON[type]);
+                        } else {
+                            return res.send(500, { errors: "Something went wrong."});
                         }
```

Comment:
This can be a conditional expression `? :`
```suggestion
                        const response = Object.keys(convertToJSON).includes(type) ?
                          res.send(convertToJSON[type]) :
                          res.send(500, { errors: "Something went wrong."});

                        return response;
```

…or something like that. If you can align the parameters, you might be able consolidate the `res.send`